web/src/app/links/ChanWebhookLink.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import AppLink from '../util/AppLink'
+
+export const ChanWebhookLink = (webhook: {
+  id: string
+  name: string
+}): JSX.Element => {
+  return <AppLink to={webhook.id}>{webhook.name}</AppLink>
```

Comment:
We can't/shouldn't link to the webhook -- it's an API target, rather than a UI landing page. We'd only put a link here if we had something like a details page within GoAlert for it.